common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -40,6 +40,7 @@
 
 class SequenceFields(object):
     has_children = True
+    completion_method = 'aggregator'
```

Comment:
Can we have a string constant somewhere?  Also, not seeing this used anywhere.

Reply:
There will be a string constant in the XBlock repo.  I'll update this to use that once it lands (possibly a later commit, depending on how long it takes to get this one merged).